src/Microsoft.ApplicationInsights.AspNetCore/Microsoft.ApplicationInsights.AspNetCore.csproj

```diff
@@ -0,0 +1,84 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <Description>Application Insights for ASP.NET Core web applications. See https://azure.microsoft.com/en-us/documentation/articles/app-insights-asp-net-five/ for more information. Privacy statement: https://go.microsoft.com/fwlink/?LinkId=512156</Description>
+    <Copyright>Copyright © Microsoft. All Rights Reserved.</Copyright>
+    <AssemblyTitle>Application Insights for ASP.NET Core Web Applications</AssemblyTitle>
+    <VersionPrefix>2.1.1</VersionPrefix>
+    <Authors>Microsoft</Authors>
+    <TargetFrameworks>net451;netstandard1.6</TargetFrameworks>
```

Comment:
Not changing anything in this PR, as this is simply converting from project.json to .csproj model.